styles/calendar_styles.css

```diff
@@ -52,4 +52,485 @@ header {
+    }
+    .jsCalendar.black-theme tbody td.jsCalendar-next:hover,.jsCalendar.black-theme tbody td.jsCalendar-previous:hover {
+        color:#FFF
+    }
+    .jsCalendar.black-theme ::-moz-selection {
+        background:#505050
+    }
+    .jsCalendar.black-theme ::selection {
+        background:#505050 
+    }
+    .jsCalendar.grey-theme table {
+        background-color:#E3E6EB;
+        box-shadow:0 0 2px rgba(0,0,0,.5)
+        ;color:#242529;
+    }
+    .jsCalendar.grey-theme thead .jsCalendar-nav-left,.jsCalendar.grey-theme thead .jsCalendar-nav-right {
+        color:#717580
+    }
+    .jsCalendar.grey-theme thead .jsCalendar-nav-left:hover,.jsCalendar.grey-theme thead .jsCalendar-nav-right:hover {
+        background-color:#717580;
+        color:#E3E6EB;
+    }
+    .jsCalendar.grey-theme tbody td.jsCalendar-selected {
+        background-color:transparent;
+        border:2px solid #A3A6AE;
+    }
+    .jsCalendar.grey-theme tbody td:hover {
+        background-color:#717580; 
+        color:#E3E6EB;
+    }
+    .jsCalendar.grey-theme tbody td.jsCalendar-current {
+        background-color:#42454C 
+    }
+    .jsCalendar.grey-theme tbody td.jsCalendar-current:hover {
+        color:#E3E6EB 
+    }
+    .jsCalendar.grey-theme tbody td.jsCalendar-next,.jsCalendar.grey-theme tbody td.jsCalendar-previous {
+        color:#B0B4BD
+    }
+    .jsCalendar.grey-theme tbody td.jsCalendar-next:hover,.jsCalendar.grey-theme tbody td.jsCalendar-previous:hover {
+        color:#E3E6EB
+    }
+    .jsCalendar.grey-theme ::-moz-selection {
+        background:#B0B4BD
+    }
+    .jsCalendar.grey-theme ::selection { 
+        background:#B0B4BD 
+    }
+    .jsCalendar.white-theme table {
+        background-color:#fff;
+        box-shadow:0 0 2px rgba(80,80,80,.4);
+        color:#333;
+    }
+    .jsCalendar.white-theme thead .jsCalendar-nav-left,.jsCalendar.white-theme thead .jsCalendar-nav-right {
+        color:#AFAFAF
+    }
+    .jsCalendar.white-theme thead .jsCalendar-nav-left:hover,.jsCalendar.white-theme thead .jsCalendar-nav-right:hover {
+        background-color:#DADADA;
+        color:#555;
+    }
+    .jsCalendar.white-theme tbody td:hover {
+        background-color:#DADADA;
+        color:#000;
+    }
+    .jsCalendar.white-theme tbody td.jsCalendar-selected {
+        border:2px solid #BFBFBF
+    }
+    .jsCalendar.white-theme tbody td.jsCalendar-current {
+        background-color:#222
+    }
+    .jsCalendar.white-theme tbody td.jsCalendar-current:hover {
+        color:#FFF
+    }
+    .jsCalendar.white-theme tbody td.jsCalendar-next,.jsCalendar.white-theme tbody td.jsCalendar-previous {
+        color:#AFAFAF
+    }
+    .jsCalendar.white-theme tbody td.jsCalendar-next:hover,.jsCalendar.white-theme tbody td.jsCalendar-previous:hover {
+        color:#000
+    }
+    .jsCalendar.white-theme ::-moz-selection {
+        background:#AFAFAF
+    }
+    .jsCalendar.white-theme ::selection {
+        background:#AFAFAF
+    }
```

Comment:
If this is all from the jsCalendar library, please move it to a separate file, or make this clear in a comment.